<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CScadaLinearScaleSetDialog</class>
 <widget class="QDialog" name="CScadaLinearScaleSetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>576</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CScadaLinearScaleSetDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QCheckBox" name="enableInverse">
     <property name="text">
      <string>EnableInverse</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>min raw</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="minRaw"/>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="4">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>max raw</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QLineEdit" name="maxRaw"/>
   </item>
   <item row="1" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>min value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="minValue"/>
   </item>
   <item row="2" column="3">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="4">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>max value</string>
     </property>
    </widget>
   </item>
   <item row="2" column="5">
    <widget class="QLineEdit" name="maxValue"/>
   </item>
   <item row="2" column="6">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QCheckBox" name="enableFactor">
     <property name="text">
      <string>EnableFactor</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="2">
    <widget class="QCheckBox" name="enableInverseFactor">
     <property name="text">
      <string>EnableInverseFactor</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>gain</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1" colspan="2">
    <widget class="QLineEdit" name="gain"/>
   </item>
   <item row="4" column="3">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="4">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>offset</string>
     </property>
    </widget>
   </item>
   <item row="4" column="5">
    <widget class="QLineEdit" name="offset"/>
   </item>
   <item row="4" column="6">
    <spacer name="horizontalSpacer_9">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>74</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <spacer name="horizontalSpacer_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>121</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="2" colspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pOk">
       <property name="text">
        <string>ok</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pCancel">
       <property name="text">
        <string>cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="5" colspan="2">
    <spacer name="horizontalSpacer_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>213</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
